MACHINE_LEARNING_ENGINEER.md

```diff
@@ -0,0 +1,71 @@
+# **Traive Challenge - Machine Learning Engineer**
+
+### **Overview**
+Below, we have outlined a problem where you will design and develop a machine learning solution to meet specific requirements. This is your opportunity to showcase your expertise! We will be assessing how you structure your project, organize your code, and apply ML concepts, best practices, and technologies to deliver a robust solution.
+
+### **Problem Statement**
+You will create a **Machine Learning Solution** in the agricultural finance domain, with a focus on end-to-end implementation from data processing to API deployment. You may choose to solve any relevant problem, such as:
+- Credit risk assessment for farms
+- Crop yield prediction
+- Agricultural commodity price forecasting
+- Farm productivity analysis
```

Comment:
This example is a bit abstract, what is the expected output here? Farm profitability? It kind of overlaps with yield prediction.